react-native/make-shim.js

```diff
@@ -18,8 +18,10 @@ try {
   fs.writeFileSync(path.join(root, 'package.json'), `{
   "main": "index.js"
 }
-`) } catch (_) {}
+`)
+} catch (_) {}
```

Comment:
Should we log error on all these try/catch failures? We weren't before so I guess not?